### PR TITLE
ResNetV1b with dilation

### PR DESCRIPTION
**Pull request by zhanghang1989**

### CODE REVIEW

**Comment by zhanghang1989** [comment]
1. ResNetV1b is ready
2. Need to retrain the FCN model before merging

**Comment by hetong007** [comment]
Please add v1b family to `gluoncv/model_zoo/model_zoo.py`: https://github.com/dmlc/gluon-cv/blob/master/gluoncv/model_zoo/model_zoo.py#L34

This enables a name-to-function matching.

**Comment by mli** [comment]
Job PR-81-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-81/7/index.html

File: tests/unittests/test_model_zoo.py [hunk 1]
```diff
@@ -49,6 +49,27 @@ def test_ssd_models():
             net.initialize()
         net(x)
 
+def test_resnet_v1b():
```
Comment (hetong007) [inline tests/unittests/test_model_zoo.py]: seems better merge with classification case.

**Comment by mli** [comment]
Job PR-81-8 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-81/8/index.html

File: tests/unittests/test_model_zoo.py (hunk 1 above)
Comment (zhanghang1989) [reply tests/unittests/test_model_zoo.py]: sure. The input size is 224x224. current classification test only seems only having cifar models

Comment (zhanghang1989) [reply tests/unittests/test_model_zoo.py]: Feel free to rename it when imagenet model test case is covered :)

**Comment by mli** [comment]
Job PR-81-9 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-81/9/index.html

**Pull request merged**